driver-core/src/main/com/mongodb/client/model/Aggregates.java

```diff
@@ -173,6 +174,26 @@ public static Bson unwind(final String fieldName) {
         return new BsonDocument("$unwind", new BsonString(fieldName));
```

Comment:
With `unwind(fieldName, options)` it would look like: `{$unwind: {$path: fieldName}}`